pkg/kubectl/cmd/util/printing.go

```diff
@@ -45,7 +45,7 @@ func AddNonDeprecatedPrinterFlags(cmd *cobra.Command) {
 	AddOutputFlags(cmd)
 	AddNoHeadersFlags(cmd)
 	cmd.Flags().Bool("show-labels", false, "When printing, show all labels as the last column (default hide labels column)")
-	cmd.Flags().String("template", "", "Template string or path to template file to use when -o=go-template, -o=go-template-file. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].")
+	cmd.Flags().String("template", "", "Template string or path to template file to use when have no -o flag, similar to -o=go-template. The template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].")
```

Comment:
`have no` -> `there is no`

Reply:
alternatively, `when no -o flag is provided`

Reply:
Thanks.